flink-tests/src/test/java/org/apache/flink/test/iterative/KMeansWithBroadcastSetITCase.java

```diff
@@ -65,7 +68,7 @@ public Centroid map(String c) {
 					}
 				});
 
-		// set number of bulk iterations for KMeans algorithm
+		// set number of bulk iterations for kmeans algorithm
```

Comment:
I reverted these and the additional instances. It looks to have happened when renaming `KMeansSingleStepTest#KMeans`.